client/src/components/Post/PostList.tsx

```diff
@@ -1,63 +1,50 @@
-import { useDispatch, useSelector } from 'react-redux';
-import allAction from '../../redux/actions';
-import { useEffect, useState } from 'react'
+import { useDispatch, useSelector } from 'react-redux'
+import allAction from '../../redux/actions'
+import { useCallback, useEffect, useState } from 'react'
 import { Post } from '../../types/Post'
 import PostItem from './PostItem'
 
 const PostList = () => {
-  const fetchedPosts = useSelector((state:any) => state.postList);
-  const dispatch = useDispatch();
-
-  useEffect(() => {
-    dispatch(allAction.loadPostList());
-  }, []);
-
   const [pageCount, setPageCount] = useState(0)
-  const [fetching, setFetching] = useState(false)
-  const [posts, setPosts] = useState<Post[]>([])
 
   const MAXIMUM_PAGES = 10
 
-  const handleScroll = () => {
-    const scrollHeight = document.documentElement.scrollHeight
-    const scrollTop = document.documentElement.scrollTop
-    const clientHeight = document.documentElement.clientHeight
+  const fetchedPosts = useSelector((state: any) => state.postList)
+  const dispatch = useDispatch()
 
+  const handleScroll = () => {
+    const { scrollHeight, scrollTop, clientHeight } = document.documentElement
     if (scrollTop + clientHeight + 100 >= scrollHeight) {
-      fetchMoreData()
+      fetchArticle()
     }
   }
 
-  window.addEventListener('scroll', handleScroll)
-  useEffect(() => {
-    return () => {
-      window.removeEventListener('scroll', handleScroll)
-    }
-  }, [])
-
-  const appendData = () => {
-    setFetching(true)
-    setPosts(fetchedPosts)
+  const fetchArticle = useCallback(() => {
+    dispatch(allAction.loadPostList(false))
```

Comment:
```suggestion
    dispatch(allAction.loadPostList(true))
```
불러오는 중이 되니까 반대가 되어야합니당